src/NHSD.BuyingCatalogue.Identity.Api/ViewModels/Account/LoginViewModel.cs

```diff
@@ -5,12 +5,12 @@ namespace NHSD.BuyingCatalogue.Identity.Api.ViewModels.Account
 {
     public sealed class LoginViewModel
     {
-        [Required]
+        [Required(ErrorMessage = "Enter your password")]
         public string Password { get; set; }
```

Comment:
We could add a data type here to represent a password. That way we don't need to specify the type within the actual html. Also I have seen this done on a few samples. Just one thing I missed when adding the initial login view.